lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -383,7 +383,7 @@ def post_data(params)
         return nil unless params
 
         params.map do |key, value|
-          next if value.nil?
+          next if value != false && value.blank?
```

Comment:
It's skipped if it's not `false` AND `blank`... It's a little counter intuitive, but the logic works:
```ruby
[1] pry(main)> value = "hello"
=> "hello"
[2] pry(main)> value != false && value.blank?
=> false
[3] pry(main)> value = false
=> false
[4] pry(main)> value != false && value.blank?
=> false
[5] pry(main)> value = ""
=> ""
[6] pry(main)> value != false && value.blank?
=> true
[7] pry(main)> value = nil
=> nil
[8] pry(main)> value != false && value.blank?
=> true
```